tests.py

```diff
@@ -188,3 +188,15 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_QtWebKitWidgets_available_in_pyside():
```

Comment:
Let's remove these tests, they aren't doing any good.

We could start testing with runnable examples, but I'm uncertain about what that would achieve also.